### PR TITLE
Fix tests in master branch

### PR DESCRIPTION
**Pull request by Denchick**
Hi!
I wanted to add PostgreSQL 14 support, but the tests in the master were red.
Due to using [the latest docker images](https://github.com/adjust/clickhouse_fdw/blob/master/docker-compose.yml#L4), some error messages and default settings in clickhouse-server changed, I updated them. @ildus Could you take a look? Could you approve me running workflows?

### CODE REVIEW

File: tests/expected/engines.out [hunk 1]
```diff
@@ -132,22 +132,22 @@ Server: loopback
 FDW options: (database 'regression', table_name 't3', engine 'MergeTree')
 
 \d+ t3_aggr
-                                              Foreign table "public.t3_aggr"
- Column |  Type   | Collation | Nullable | Default |         FDW options          | Storage  | Stats target | Description 
---------+---------+-----------+----------+---------+------------------------------+----------+--------------+-------------
- a      | integer |           | not null |         |                              | plain    |              | 
- b      | istore  |           | not null |         | (aggregatefunction 'sumMap') | extended |              |
```
Comment (Denchick) [inline tests/expected/engines.out:139]: This is the only place which I'm not sure about.

File: tests/expected/engines.out [hunk 2]
```diff
@@ -132,22 +132,22 @@ Server: loopback
...
 Server: loopback
 FDW options: (database 'regression', table_name 't3_aggr', engine 'MaterializedView')
 
 \d+ t4
-                                                  Foreign table "public.t4"
- Column |  Type   | Collation | Nullable | Default |           FDW options           | Storage  | Stats target | Description 
---------+---------+-----------+----------+---------+---------------------------------+----------+--------------+-------------
- a      | integer |           | not null |         |                                 | plain    |              | 
- b      | integer |           | not null |         | (aggregatefunction 'sum')       | plain    |              | 
- c      | istore  |           | not null |         | (aggregatefunction 'sumMap')    | extended |              |
```
Comment (Denchick) [inline tests/expected/engines.out:149]: And this.

**Comment by ildus** [comment]
@Denchick sorry, I don't have access to repo anymore

**Comment by Denchick** [comment]
Maybe you can mention someone who can review the PR?

**Comment by Denchick** [comment]
@za-arthur, can you take a look? Or maybe do you know somebody who can?

**Comment by za-arthur** [comment]
Sorry I cannot say definitely about plans to support this for now.

**Comment by Denchick** [comment]
Screenshot 2022-02-28 at 21 33 23

**Comment by za-arthur** [comment]
@Denchick thank you for the PR! I'm merging it.

**Pull request merged**